docs/getting_support.rst

```diff
@@ -254,34 +254,33 @@ from someone in your organization, we may assign it back to you.
 
 Encrypted Email
 ---------------
-
-One of Redmine's strengths as a ticketing system is its powerful support
-for email-based workflow. You can use email to create new issues, reply
-to existing issues, and be notified of updates to issues that are
-relevant to you.
-
 While many people find email-based workflows convenient, email is
 unfortunately insecure by default. Freedom of the Press Foundation takes
 the security of every SecureDrop instance seriously; therefore, we
 require the use of encryption for support requests because they may
 contain sensitive information about your SecureDrop instance.
 
-The web interface workflow is automatically encrypted thanks to HTTPS.
-Supporting a secure email-based workflow is more difficult because email
-is unencrypted by default. Our solution is to combine Redmine's
-email-based workflow with OpenPGP encryption, which we already
-use to communicate with many SecureDrop administrators and journalists.
+After setting up your account, you will have the option of enabling
+encrypted email.
+
+There are some several benefits to enabling encrypted email, such as:
 
-What if I don't want to use encrypted email?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* receiving the full contents of messages and notifications
+* the ability to create new issues by emailing support@freedom.press
```

Comment:
Can we double check what happens to unencrypted emails to support@?

Reply:
Replying to an encrypted notification email in plaintext _does_ indeed work and logs the ticket response as expected. Also, emailing support@ directly still files a new issue. I have removed those two bullet points and reworked the paragraph.